samples/SampleApp/project.json

```diff
@@ -1,12 +1,25 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "Microsoft.AspNetCore.Http": "1.0.0-*"
-  },
-  "commands": {
-    "SampleApp": "SampleApp"
+    "Microsoft.AspNetCore.Http": "1.0.0-*",
+    "Microsoft.AspNetCore.Http.Extensions": "1.0.0-*"
   },
   "frameworks": {
-    "net451": {}
+    "net451": { },
+    "netcoreapp1.0": {
+      "imports": [
+        "dnxcore50"
+      ],
+      "dependencies": {
+        "System.Console": "4.0.0-*",
```

Comment:
Why do you need this? I think it's pulled in via `Microsoft.NETCore.App` via `NETStandard.Library`.